enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1321,6 +1325,70 @@ def test_credits_available_endpoint(
             # with an inactive (i.e., expired, not yet started) subsidy, we should get no records back.
             assert len(response_json) == 0
 
+    @mock.patch('enterprise_access.apps.subsidy_access_policy.models.get_and_cache_transactions_for_learner')
+    @mock.patch('enterprise_access.apps.subsidy_access_policy.models.SubsidyAccessPolicy.subsidy_record')
+    def test_credits_available_endpoint_with_content_assignments(
+        self,
+        mock_subsidy_record,
+        mock_transactions_cache_for_learner  # pylint: disable=unused-argument
+    ):
+        """
+        Verify that SubsidyAccessPolicyViewset credits_available returns learner content assignments for assigned
+        learner credit access policies.
+        """
+        content_key = 'demoX'
+        content_title = 'edx: Demo 101'
+        content_price_cents = 100
+        # Create a pair of AssignmentConfiguration + SubsidyAccessPolicy for the main test customer.
+        assignment_configuration = AssignmentConfigurationFactory(
+            enterprise_customer_uuid=self.enterprise_uuid,
+        )
+        AssignedLearnerCreditAccessPolicyFactory(
+            display_name='An assigned learner credit policy, for the test customer.',
+            enterprise_customer_uuid=self.enterprise_uuid,
+            active=True,
+            assignment_configuration=assignment_configuration,
+            spend_limit=1000000,
+        )
+        assignment1 = LearnerContentAssignmentFactory.create(
+            assignment_configuration=assignment_configuration,
+            learner_email='alice@foo.com',
+            lms_user_id=1234,
+            content_key=content_key,
+            content_title=content_title,
+            content_quantity=-content_price_cents,
+            state=LearnerContentAssignmentStateChoices.ALLOCATED,
+        )
+        LearnerContentAssignmentFactory.create(
```

Comment:
Maybe also worth adding a `LearnerContentAssignmentAction` record into the mix, too.